### ISSUE TITLE
TypeError: Expected str, got float

### ISSUE THREAD

**Issue by jingyulong** [Roche/pyreadstat#54]
# Describe the issue

When I use ```pyreadstat.write_sav``` to save the file, run the python script file in the terminal, this error pops up directly: ```TypeError: Expected str, got float```, and then the program ends. I used ```Try expect```, and it will skip it. I can't find any information that can be debugged. Is there any good way to debug this problem?

# Expected behavior

When an exception occurs, it can prompt the corresponding information, such as the wrong value and type

# Setup Information:

### How did you install pyreadstat? (pip, conda, directly from repo)

pip  

### pyreadstat version

0.3.1

### Platform (windows, macOS, linux, 32 or 64 bit)

windows

### Python Version

3.7

### Python Distribution (System, plain python, Anaconda)

Anaconda

**Comment by ofajardo** [comment]
Please provide a way to reproduce the issue, otherwise it's impossible to help.

**Comment by jingyulong** [comment]
> It is an unexpected error, so no information can be supplied in advance.
> 
> Please provide a way to reproduce the issue, otherwise it's impossible to help.

Thanks for the reply, I wrote a small piece of code to reproduce the problem

```
import numpy as np
import pandas as pd
import pyreadstat

df=pd.DataFrame([[0,np.nan,],[1,"test"]])

pyreadstat.write_sav(df,"test.sav")
```

**Comment by ofajardo** [comment]
Thanks! I can reproduce it, I will take a look and fix it

**Comment by ofajardo** [comment]
issues solved in version 0.3.2
